pkg/cmd/pr/shared/commentable.go

```diff
@@ -262,7 +269,7 @@ func deleteComment(commentable Commentable, opts *CommentableOptions) error {
 
 	if opts.IO.CanPrompt() && !opts.DeleteLastConfirmed {
 		fmt.Fprintf(opts.IO.Out, "%s Deleted comments cannot be recovered.\n", cs.WarningIcon())
-		ok, err := opts.ConfirmDeleteLastComment(lastComment.Body)
+		ok, err := opts.ConfirmDeleteLastComment(text.Truncate(40, lastComment.Body))
```

Comment:
This doesn't work for our use case because it just truncates long lines and not the string as a whole. The best is to do an ordinary slice range indexing. I know it's not the best way to do it, but I think it's okay.